2025.md

```diff
@@ -68,3 +68,6 @@
 * [🌍Asgardeo’s Global Footprint: How Its Architecture Simplifies Multi-Region Deployment](https://medium.com/@jayanga/asgardeos-global-footprint-how-its-architecture-simplifies-multi-region-deployment-6929c81ff451) by [Jayanga Kaushalya](https://medium.com/@jayanga)
 * [Proof of Work vs Proof of Stake: What's the Difference?](https://medium.com/@jayashakthiperera/proof-of-work-vs-proof-of-stake-whats-the-difference-51f539f5cd91) by [Jayashakthi Perera](https://medium.com/@jayashakthiperera)
 * [What is Web3? A Simple Explanation for Software Engineers](https://medium.com/@jayashakthiperera/what-is-web3-a-simple-explanation-for-software-engineers-9a8b79bc096e) by [Jayashakthi Perera](https://medium.com/@jayashakthiperera)
+
+## November (1)
+* [Sharing the Load: Clustering WSO2 Identity Server Locally](https://bodaragamadb-office.medium.com/sharing-the-load-clustering-wso2-identity-server-locally-c9897fd7786b) by [Bimsara Bodaragama](https://bodaragamadb-office.medium.com/)
```

Comment:
Let's add your name and medium link in bloggers list as well
https://github.com/wso2/iam-blogs/blob/main/README.md